voting/src/voting.rs

```diff
@@ -207,6 +207,27 @@ impl TxOwner for Context {
                 }
                 Ok(())
             }
+            VOTE_TX_TYPE => {
+                let tx: VoteTransaction = serde_cbor::from_slice(&transaction.body()).map_err(|_| todo_fixthis)?;
+                let vote_paper_id = tx.action.vote_paper_id;
+                let choice = tx.action.choice;
+                let voter_signature = tx.action.voter_signature;
+
+                if choice != VoteChoice::Favor && choice != VoteChoice::Against && choice != VoteChoice::Absention {
+                    return Err(ExecuteError::InvalidChoice).map_err(|_| todo_fixthis)
+                }
```

Comment:
Let's remove this code.

We are using enum for the VoteChoice. If the choice is not one of Favor, Against, and Absention, there will be a deserialization error.